RecoLocalTracker/SiPhase2VectorHitBuilder/python/SiPhase2RecHitMatcher_cfi.py

```diff
@@ -0,0 +1,17 @@
+import FWCore.ParameterSet.Config as cms
+
+SiPhase2RecHitMatcherESProducer = cms.ESProducer("SiPhase2RecHitMatcherESProducer",
```

Comment:
Added a fillDescriptions method

Reply:
Then this file can be removed, no?

Reply:
I tried that, but it resulted in errors because of this import https://github.com/cms-sw/cmssw/blob/08a28ed68b41292bbca9dee719d48b4135ca31e4/RecoLocalTracker/Configuration/python/RecoLocalTracker_cff.py#L26

Reply:
Then I'd suggest to fix the import to use `...SiPhase2RecHitMatcherESProducer_cfi`. The convention for cfi files anyway is to have the name of the defined object in the file (as is to start with lowercase letter; and yes, there is tons of old stuff breaking either or both).

Reply:
For some reason the import still doesn't work. I can see that <siPhase2RecHitMatcherESProducer_cfi.py> is created in <../cfipython/slc7_amd64_gcc820/RecoLocalTracker/SiPhase2VectorHitBuilder/>, but the import <from RecoLocalTracker.SiPhase2VectorHitBuilder.siPhase2RecHitMatcherESProducer_cfi import *> still results in <ImportError: No module named siPhase2RecHitMatcherESProducer_cfi> Am I missing something?

Reply:
The file is now removed.